runtime/win32.c

```diff
@@ -607,6 +607,11 @@ void caml_win32_overflow_detection(void)
   AddVectoredExceptionHandler(1, caml_stack_overflow_VEH);
 }
 
+void caml_win32_unregister_overflow_detection(void)
+{
+  RemoveVectoredContinueHandler(caml_stack_overflow_VEH);
```

Comment:
Hang on, shouldn't this `RemoveVectoredExceptionHandler`??

Reply:
> Hang on, shouldn't this be `RemoveVectoredExceptionHandler`??

Exactly. I've already fixed that locally. I'll push the updated commit on Monday.
Funny thing is that I tested all this before creating the PR, and it worked somehow... Sorry again.